Binance.Net/Objects/BinanceFuturesPosition.cs

```diff
@@ -13,34 +14,42 @@ public class BinanceFuturesPosition
         /// <summary>
         /// The entry price of the position
         /// </summary>
+        [JsonProperty("entryPrice")]
```

Comment:
Is there a specific reason you added these attributes? Json.Net will map them regardless of capitalization normally.

Reply:
There is no specific reason